src/code/z_kanfont.c

```diff
@@ -1,7 +1,15 @@
 #include "global.h"
 #include "message_data_static.h"
 
-void func_8006EE50(Font* font, u16 arg1, u16 arg2) {
+/**
+ * Loads a texture from kanji for the requested `character` into the character texture buffer
+ * at `codePointIndex`. The value of `character` is the SHIFT-JIS encoding of the character.
+ */
+void Font_LoadKanji(Font* font, u16 character, u16 codePointIndex) {
```

Comment:
why would "linguists be mad"? does this technically have hiragana and katakana too?

Reply:
The Kanji file has every glyph needed to render SHIFT-JIS decoded text. This includes hiragana, katakana, and more.

I suggest `Font_LoadGlyphFromKanji` to indicate more explicitly that it handles loading from the kanji file, rather than that it exclusively loads kanji glyphs.

Reply:
`Font_LoadGlyphFromKanji` is not bad, but I don't like that it's not consistent with `Font_LoadChar`. How about something like `Font_LoadCharWide`?

Reply:
Sounds good to me

Reply:
alright made the change, @fig02 let me know what you think

Reply:
Personally feel like this is way less helpful information-- with the word "kanji" I at least know its going to be japanese related. "wide" doesn't really mean much to me in this context.
But if others are fine with it I will get used to it

Reply:
yeah, tbh I'm not sold on "wide" vs "non-wide" either. Maybe something like "Japanese" vs "Latin"(?) would be better. But at least I think everything consistently uses wide/non-wide now, so maybe it's fine for now